src/assets/styles.css

```diff
@@ -0,0 +1,17 @@
+.custom-css wizard-navigation-bar.horizontal ul.steps-indicator li.current div a {
```

Comment:
Is it possible to move the whole content of this file, to `src/app/custom-css/custom-css.component.css`?

Reply:
No problem.